### DISCUSSION TITLE
Feature Request: Sorting of Issues

### DISCUSSION THREAD

**Discussion by rjonesy** [community/community#15] [Mobile]
_Y’all’s work on GitHub mobile inspires me – your speed, attention to customer-based outcomes, and willingness to try ideas is something I strive to mimic. Thanks for fueling me and for putting your great work out!_

Currently: The `Issues` page appears to be sorted by `Date Created`. 

Context: We use Issues to document development tasks, with ongoing comments for progress. For example, I’ll learn something new and comment on the Issue. Typically what happens is 2-4 bigger Issues are being discussed at once, while smaller issues that don’t need discussion are just Completed and pushed. The result is continuous comments on a few Issues and ~no comments on most issues. The “problem” is when an active Issue was created a while ago, I have to search or scroll to find it each time – versus my expectation that recently changed/modified Issues would be towards the top of the sort. 

Proposed: It could be desirable to add a Sort icon to the left of the + icon, with a popover (?) for the different options. This leaves room for higher fidelity control, and expansion into Filtering if you may want that too. In the short term, another possibility is `Sort by Modified`. No additional buttons needed. It would just be a default state. There could be downsides but I can’t think of them. 

[Here’s a quick video](https://www.dropbox.com/s/tfpiapujs4dsgmc/RPReplay_Final1602951159.mp4?dl=0), showing an interaction I do 10x+ per day and why the current implementation isn’t ideal. 

Front.app has a great implementation of this, and the icon changes to match your current sort type. 
![image](https://user-images.githubusercontent.com/5297381/96375002-eead2780-113b-11eb-8388-ae4f9a2d6e4d.png)

**Comment by candyho** [comment]
Hey @rjonesy , thanks for your kind words and sharing your suggestion on Issues sorting. The suggestion looks great, I've pass it along to our team and we'll keep monitoring and improving this space.
